chrome/background.html

```diff
@@ -0,0 +1,4 @@
+<html>
+    <script type="text/javascript" src="background.js"></script>
+    <script type="text/javascript" src="vendor/polyfill.min.js"></script>
```

Comment:
This needs to be before background.js, so it has a chance to polyfill stuff before that runs